src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,49 +1,70 @@
+        <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
+        <br> If you have any remaining questions, don't hesitate to <a href="mailto:{{ supportEmail }}">email us</a>. We respond within 24 hours.
       </p>
-      <ul>
-        <li>
-          <a href="#students">Students</a>
-        </li>
-        <li>
-          <a href="#courses">Courses</a>
-        </li>
-        <li>
-          <a href="#sessions">Sessions</a>
-        </li>
-        <li>
-          <a href="#questions">Questions</a>
-        </li>
-      </ul>
-    </div>
-  </div>
-  <div class="row">
-    <div class="offset-md-2 col-md-8">
-      <div class="input-group input-group-lg">
-        <input type="text"  [(ngModel)]="searchTerm" class="form-control input-lg" placeholder="How can we help?" (keyup.enter)="search()">
-        <button class="btn btn-primary btn-lg" (click)="search()"><i class="fas fa-search"></i></button>
-        <button class="btn btn-danger btn-lg" (click)="clear()"><i class="fas fa-times"></i></button>
+      <div class="row">
+        <div class="offset-md-2 col-md-8">
+          <div class="input-group input-group-lg">
+            <input type="text" [(ngModel)]="searchTerm" class="form-control input-lg" placeholder="How can we help?" (keyup.enter)="search()">
+            <button class="btn btn-primary btn-lg" (click)="search()"><i class="fas fa-search"></i></button>
+            <button class="btn btn-danger btn-lg" (click)="clear()"><i class="fas fa-times"></i></button>
+          </div>
+        </div>
       </div>
+      <hr/>
     </div>
   </div>
-  <hr/>
-  <div>
-    <tm-instructor-help-students-section [key]="key">
+  <div id="students">
+    <tm-instructor-help-students-section [key]="key"> 1
     </tm-instructor-help-students-section>
-    <tm-instructor-help-sessions-section [key]="key"></tm-instructor-help-sessions-section>
-    <tm-instructor-help-questions-section [key]="key"></tm-instructor-help-questions-section>
   </div>
-  <div>
-    <tm-instructor-help-courses-section [key]="key">
+  <div id="courses">
+    <tm-instructor-help-courses-section [key]="key"> 2
     </tm-instructor-help-courses-section>
   </div>
+  <div id="sessions">
+    <tm-instructor-help-sessions-section [key]="key"> 3
+    </tm-instructor-help-sessions-section>
+  </div>
+  <div id="questions">
+    <tm-instructor-help-questions-section [key]="key"> 4
+    </tm-instructor-help-questions-section>
+  </div>
+</div>
+
+<div id="topics" class="side-nav container-fluid col-sm-3 col-4">
+  <p>
+    Browse by topic:
+    <br>
+  </p>
+  <ul class="nav nav-pills flex-column">
+    <li class="nav-item">
+      <a (click)="scroll('students')" [routerLink]="">Students</a>
+      <ng-container *ngIf="currentSection === 'students'"></ng-container>
+    </li>
+    <li class="nav-item">
+      <a (click)="scroll('courses')" [routerLink]="">Courses</a>
```

Comment:
Shall we convert these parameter strings ('courses', 'sessions') etc as ENUMS instead so that it facilitates maintenance and adding future new sections? Same for the rest.

Reply:
Sure I'll do that

Reply:
Done!